### PR TITLE
StickyEvents : Adding methods to fire when this.mode is changed

### LINKED ISSUE #13: Detecting sticky section is applied
Hi,
I need to call a function when the section is sticky. Is there any way to detect whenever we have a sticky section at the top/bottom of a page?

### PR DESCRIPTION
**Pull request by sepehr1313**
Closes #13

### CODE REVIEW

**Comment by sepehr1313** [comment]
Thanks for the feedback. I added a function named onChangeMode(oldMode, newMode). Hope that works

File: src/index.js [hunk 1]
```diff
@@ -184,3 +191,11 @@ export default class StickyBox extends React.Component {
     );
   }
 }
+
+StickyBox.defaultProps = {
+  onChangeMode: () => {},
```
Comment (danielberndt) [inline src/index.js]: there already is a `defaultProps` function in line 31,
it probably makes sense to move this, as well as the propTypes definition up there. (Feel free to add PropType definitions for `bottom: boolean, and offset: number` there as well :)

**Comment by danielberndt** [comment]
Looking good! I'll publish it in a minute

**Comment by danielberndt** [comment]
Just added it to the Readme, it's now live with version 0.6.4

**Pull request merged**